package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cjaas/common-components",
-  "version": "1.0.0",
+  "version": "1.2.0",
```

Comment:
why not 1.1.0?

Reply:
Actually I believe it will be 2.0.0
1.1.0 is where our dev deploys are currently, and these changes will progress beyond them